juju_verify/verifiers/base.py

```diff
@@ -46,19 +46,26 @@ class BaseVerifier:
 
     NAME = ""
 
-    def __init__(self, units: List[Unit]):
+    def __init__(
+        self, units: List[Unit], exclude_affected_units: Optional[List[Unit]] = None
+    ):
         """Initiate verifier linked to the Juju units.
 
         All the checks that the verifier implements must expect that the action
         that is being verified is intended to be performed on all juju units
         in the 'self.units' simultaneously.
 
+        Parameter `exclude_affected_units` should contain list of units that are
+        explicitly checked by other Verifier(s) so that they can be excluded from
+        the warnings in `check_affected_machines()`.
```

Comment:
IMO this could be used in any another check, so this comment could be changed?

Reply:
How about this
```
so that they can be excluded from the warnings in `check_affected_machines()` or other charm-specific checks.
```
I'd like to keep reference to the `check_affected_machines` because it gives reference for how it's actually used.

Reply:
That's perfect.